Chaincase.Common/Extensions.cs

```diff
@@ -16,21 +18,26 @@ public static class Extensions
     {
         public static IServiceCollection AddCommonServices(this IServiceCollection services)
         {
+	        services.AddOptions()
+		        .ConfigureOptions<Config>()
+		        .PostConfigure(new Action<Config>(config =>
+		        {
+			        config.MixUntilAnonymitySet = Config.GetNormalizeAnonSet(config);
```

Comment:
This should load the file from disk as soon as a IOptions<Config> is required, right?
What if DataDir doesn't exist yet?